include/swupd.h

```diff
@@ -117,6 +117,7 @@ struct file {
 	unsigned int is_file : 1;
 	unsigned int is_link : 1;
 	unsigned int is_deleted : 1;
+	unsigned int is_ghosted : 1;
```

Comment:
would this be the right place to document what a "ghost" file means?

Reply:
Sure, it doesn't hurt to add a comment here if that's what you mean.